packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -179,16 +215,18 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
             }
         });
     }
-
+    /* tslint:disable */
```

Comment:
Can we specify what we are disabling here? This will wholesale disable. If we need to disable an instance temporarily, let's be sure we specify which one rather than disable tslint altogether.